src/types/types.ts

```diff
@@ -141,6 +141,9 @@ export type ParticipantAccountSummary = {
   externalIds: StringDictionary<string>
   studyExternalId?: string
   status: 'unverified' | 'pending' | 'verified'
+  externalId?: string
```

Comment:
I don't think we need this here. This should map to what we get from the API

Reply:
Gotcha. It appears that some results from the API do have an externalID field while others do not:
![Screen Shot 2021-03-04 at 9 24 18 AM](https://user-images.githubusercontent.com/31671708/110003623-68007d80-7ccb-11eb-81a5-e162327aeec6.png)
v.s.
![Screen Shot 2021-03-04 at 9 24 40 AM](https://user-images.githubusercontent.com/31671708/110003672-751d6c80-7ccb-11eb-8c82-df57570158e4.png)

I will remove it.